.drone.yml

```diff
@@ -14,9 +21,15 @@ pipeline:
       - yarn install
       - yarn build:system
 
-  publish:
+  publish-docs:
     image: plugins/gh-pages:1
     secrets: [github_username, github_password]
     pages_directory: dist/docs
     when:
       event: [push]
+
+  publish-system:
+    image: plugins/npm:1
+    secrets: [npm_username, npm_email, npm_token]
+    when:
+      event: [push]
```

Comment:
Also on tag, right?

Reply:
yes - I'll change this once we start tagging - I need to create a different version then in package.json